### PR TITLE
add file_id to FileStats response

### PR DESCRIPTION
**Pull request by boddumanohar**
### Changes
- add `file_id` to fileStats response

### Fixes
-

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- blobber: https://github.com/0chain/blobber/pull/952
- 0chain:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: zboxcore/fileref/fileref.go [hunk 1]
```diff
@@ -76,6 +76,7 @@ type Ref struct {
 	Children         []RefEntity      `json:"-" mapstructure:"-"`
 	CreatedAt        common.Timestamp `json:"created_at" mapstructure:"created_at"`
 	UpdatedAt        common.Timestamp `json:"updated_at" mapstructure:"updated_at"`
+	AllocationRoot   string           `json:"WriteMarker"`
```
Comment (dabasov) [inline zboxcore/fileref/fileref.go]: `json:"allocation_root"`

**Review by dabasov — CHANGES_REQUESTED** [review]

File: zboxcore/fileref/fileref.go (hunk 1 above)
Comment (boddumanohar) [reply zboxcore/fileref/fileref.go]: https://github.com/0chain/blobber/blob/3c0de5ff2f6732dfd1a7f3a0bf73e4d0a7ba3476/code/go/0chain.net/blobbercore/allocation/file_changer_upload.go#L91

https://github.com/0chain/blobber/blob/2e2fe75f2b7de1e2c96f4e25564f776cc0321c01/code/go/0chain.net/blobbercore/allocation/file_changer_update.go#L90

https://github.com/0chain/blobber/blob/3c0de5ff2f6732dfd1a7f3a0bf73e4d0a7ba3476/code/go/0chain.net/blobbercore/allocation/copyfilechange.go#L134


on the blobber, this is actually allocationRoot

Comment (boddumanohar) [reply zboxcore/fileref/fileref.go]: https://github.com/0chain/blobber/blob/aa49a30b53415f01e00b36c6e7f814d265ed35dc/code/go/0chain.net/blobbercore/reference/ref.go#L49
`WriteMarker` in Ref table is actually allocation root

Comment (boddumanohar) [reply zboxcore/fileref/fileref.go]: cc: @lpoli

**Comment by boddumanohar** [comment]
https://0chain.slack.com/archives/G014PQ61WNT/p1676968199633019?thread_ts=1676710376.021699&cid=G014PQ61WNT

instead of allocation_root, `file_meta_root` is a more useful field to have 

on hold until https://github.com/0chain/0chain/pull/1791 and the associated PRs are merged.

**Comment by boddumanohar** [comment]
updated instead of allocation_root, added `file_id` field.

File: zboxcore/fileref/fileref.go [hunk 2]
```diff
@@ -73,7 +73,7 @@ type Ref struct {
 	NumBlocks        int64  `json:"num_of_blocks" mapstructure:"num_of_blocks"`
 	PathHash         string `json:"path_hash" mapstructure:"path_hash"`
 	LookupHash       string `json:"lookup_hash" mapstructure:"lookup_hash"`
-	FileID           string `json:"file_id" mapstructure:"file_id"`
+	FileID           string `json:"FileID" mapstructure:"file_id"`
```
Comment (dabasov) [inline zboxcore/fileref/fileref.go]: lets keep all json tags in spinal case

**Review by dabasov — CHANGES_REQUESTED** [review]

File: zboxcore/fileref/fileref.go (hunk 2 above)
Comment (boddumanohar) [reply zboxcore/fileref/fileref.go]: fixed

**Comment by boddumanohar** [comment]
output after running: `zbox stats --allocation $alloc --remotepath /go12.sum --json --silent | jq .`

```
{  
  "aece9ef20a850a00c5018f09c3bcdc7935b685cc1e7bcd86549325cf6b5dd415": {
    "name": "go12.sum",
    "size": 66308,
    "path_hash": "b2b37032d1abd25d386a8fe57cd917b9736c762c7480c1ea9dcd78f1e1134602",
    "path": "/go12.sum",
    "num_of_blocks": 2,
    "num_of_updates": 1,
    "num_of_block_downloads": 0,
    "num_of_challenges": 0,
    "num_of_failed_challenges": 0,
    "last_challenge_txn": "",
    "write_marker_txn": "7d7ae03b2867092d009c5d901d7774f2723033454708064f4f33844ad256b292",
    "blobber_id": "aece9ef20a850a00c5018f09c3bcdc7935b685cc1e7bcd86549325cf6b5dd415",
    "blobber_url": "https://demo1.zus.network/blobber02",
    "blockchain_aware": true,
    "CreatedAt": "0001-01-01T00:00:00Z",
    "file_id": "21959aff-faa2-5db0-8391-4fdfcf074178"
  }
}
```

**Review by dabasov — COMMENTED** [review]
lgtm

**Review by dabasov — APPROVED** [review]

**Pull request merged**